api/v1alpha2/zz_generated.conversion.go

```diff
@@ -958,6 +958,7 @@ func autoConvert_v1alpha3_MachineSpec_To_v1alpha2_MachineSpec(in *v1alpha3.Machi
 	out.InfrastructureRef = in.InfrastructureRef
 	out.Version = (*string)(unsafe.Pointer(in.Version))
 	out.ProviderID = (*string)(unsafe.Pointer(in.ProviderID))
+	// WARNING: in.FailureDomain requires manual conversion: does not exist in peer-type
```

Comment:
We need to add this to the conversion webhook, making sure we don't lose the field when down-converting and then back up. There is an example and tests in conversion.go files, lemme know if you need more pointers

Reply:
ah, no problems.

what's the situation where we down convert then up convert?

Reply:
updated!